README.md

```diff
@@ -1,60 +1,67 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# JavaScript Capstone - JS Meals
 
-> One paragraph statement about the project.
+> This is a website for showing dishes and interacting with them.
 
-![screenshot](./app_screenshot.png)
+![screenshot](./website.png)
 
-Additional description about the project and its features.
+In this website we are interacting with two different APIs not only to show the meals but also to let users like them and/or comment about them.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- JavaScript
+- HTML&CSS
+- Bootstrap
+- Webpack
+- Git & Github
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://jmontanholi.github.io/JavaScriptCapstone-joao-sebastian/)
 
```

Comment:
- [ ] I would recommend adding a video presentation in which :wink: 
 
 - you can talk about the features of the project.
 - The maximum length of the video is 5 minutes.
 - The video must contain a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
 - All the members in the video should appear in the video and talk about the project.
 - It should contain a highlight about a piece of the code the as group you find interesting or is proud of making.